### ISSUE TITLE
Attempting to open a .DSK that is read-only results in an error

### ISSUE THREAD

**Issue by tomcw** [fadden/CiderPress2#9]
I'm using: CiderPress II v1.0.0

If I right-click the .DSK and make it read-only then I get this error:

```
---------------------------
File error
---------------------------
Error: Access to the path 'C:\Users\tomch\Downloads\mb-audit.dsk' is denied.
---------------------------
OK   
---------------------------
```

Unchecking the read-only file property allows me to open the file.

**Comment by fadden** [comment]
Looks like I'm trapping the wrong exception (should be UnauthorizedAccessException rather than IOException) in the "retry as read-only" code. Easy to fix.

**Comment by fadden** [comment]
Fix is available in https://github.com/fadden/CiderPress2/releases/tag/v1.0.1-dev1

**Comment by fadden** [comment]
Available in v1.0.1.
